packages/webpack/src/buildtime/wrapper.js

```diff
@@ -121,6 +121,6 @@ ${e}
         `
       )
     })
-    throw Error(validityFlag + 'wrapped module is not valid JS\n' + e)
+    throw Error(validityFlag + ': wrapped module is not valid JS\n' + e)
```

Comment:
```suggestion
    throw Error(validityFlag + ': wrapped module is not valid JS\n', {cause: e})
```

Reply:
The way these are processed I'm not sure the cause would be surfaced to the user.